.asf.yaml

```diff
@@ -62,12 +62,6 @@ github:
     merge: false
     rebase: false
 
-  pull_requests:
-    # allow auto-merge
-    allow_auto_merge: true
-    # auto-delete head branches after being merged
-    del_branch_on_merge: true
-
```

Comment:
We also need remove the "required_status_checks" below

Reply:
Fixed